modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties

```diff
@@ -1,3 +1,5 @@
+css-urls=CSS URL(s)
```

Comment:
Did you do buildLang?

Reply:
I never do that. I though it was Brian who did that... :thinking:

Reply:
Hehehe, we usually do it (Echo does it too) but tbh I'm not so sure if we are stepping into Brian's toes.

Reply:
I've never done it because I don't have the credentials. I had them, but since long ago we were told not to run it (and Brian has never complaint about it).